src/app/workspace/resource/representation/add-region-form/add-region-form.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+@Component({
+    selector: 'app-add-region-form',
+    templateUrl: './add-region-form.component.html',
+    styleUrls: ['./add-region-form.component.scss']
+})
+export class AddRegionFormComponent implements OnInit {
+    @Input() resourceIri: string;
+    regionForm: FormGroup;
+    colorPattern = '^#[a-f0-9]{6}$';
+    constructor(private _fb: FormBuilder) {
+    }
+
+    ngOnInit(): void {
+        this.regionForm = this._fb.group({
+            color: ['#ff3333', [Validators.required, Validators.pattern(this.colorPattern)]],
+            comment: [null, Validators.required],
```

Comment:
is a comment actually required to submit the region?

Reply:
Yes, otherwise the api is not happy. Not sure why though.